src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -148,21 +152,22 @@ public void testBQSRBucket(BQSRTest params) throws IOException {
     public void testPlottingWorkflow() throws IOException {
```

Comment:
i think @jean-philippe-martin switched to a different input file for the cloud because of the reference. but the expected output was not updated. #1118 will fix that